.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+Fixes #
+
+Checklist
+ []-I have read the Contribution & Best practices Guide and my PR follows them.
+ []-My branch is up-to-date with the Upstream master branch.
+ []-The unit tests pass locally with my changes
+ []-I have added tests that prove my fix is effective or that my feature works
+ []-I have added necessary documentation (if appropriate)
+ []-All the functions created/modified in this PR contain relevant docstrings.
+
+Test Passing
+[]- The SUSI Server must be building on the pi on bootup
+[]- The hotword detection should have a decent accuracy
+[]- SUSI Linux shouldn't crash when switching from online to offline and vice versa(failing as of now)
```

Comment:
Please don't let the open brace `(` stick to there previous word.

`versa(failing` looks ugly. Fix: `versa (failing`.